src/ui/qgstextformatwidgetbase.ui

```diff
@@ -2732,7 +2732,7 @@ font-style: italic;</string>
                               <string>Size</string>
                              </property>
                              <property name="alignment">
-                              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+                              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
```

Comment:
Editing .ui files with your favorite text editor is often much easier and less disruptive than via QtCreator :)